### DISCUSSION TITLE
Easiest setup using GitHub Gist

### DISCUSSION THREAD

**Discussion by kissge** [marp-team/marp#276] [Ideas]
This is just a random idea, so I'm not sure many will like it.

Anyway, if you can setup Marp slides just by uploading a Markdown file on GitHub Gist and then accessing a certain webpage, wouldn't that be great?
For example: I'd upload a Markdown file on https://gist.github.com/kissge/xxxxxxxxxxxxx, then I can present my Marp slides just by opening https://gist.marp.app/kissge/xxxxxxxxxxxxx.

This wouldn't be too difficult to implement, so anyone (or I) can make it by themself, but I think it's best if provided by the official website.

I actually found something quite similar but with reveal.js: https://gistdeck.github.io/#19

**Comment by yhatt** [comment]
Our goal is for making the open-sourced presentation ecosystem. It means we are focusing for making the framework and base system for building slides. So every developers can try to create the GitPitch-like integration using Gist and Marp (Marp Core).

On the other hand, we have no concrete plans to work as Marp team for integration with other tools. Currently we have no times to maintain Web service together with our ecosystem. We believe the open source community to extend Marp's power. :)
